simbad/lattice/latticesearch.py

```diff
@@ -85,6 +99,7 @@ def search(self, space_group, unit_cell, tolerance=0.05, max_to_keep=50, max_pen
         with numpy.load(self.lattice_db_fname) as compressed:
             for entry in compressed["arr_0"]:
                 pdb_code = "".join(chr(c) for c in entry[:4].astype('uint8'))
+                pdb_path = os.path.join(self.working_dir, 'mr_models', '{0}.pdb'.format(pdb_code))
```

Comment:
Not sure this is a good idea. Could we not provide the directory path in the `__init__` function?

Reply:
I essentially only moved where the pdb_path was created from the mr_module to the lattice search module. Tbh this is a bit of a hack to ensure that the pdb_path was input into the mr module the same way as in the amore_search. I could probably change working_dir to model_dir, is that what you're suggesting here?

Reply:
I'm not entirely sure I follow. Basically, I think we should avoid hard-coding the path and instead let that be something that comes from outside of this class, basically where we define `os.path.join(self.working_dir, 'mr_models')`.